app/models/repository.rb

```diff
@@ -21,6 +22,11 @@ class Repository < ActiveRecord::Base
     # options :namespace_name, type: :fulltext
   end
 
+  # Returns the number of Comments
+  def number_of_comments
```

Comment:
`comments.all.count` is the same as `comments.count`. So, remove this method altogether.